docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+
+## Deploying on Mumbai
+
+Before running this command, you should customize the parameters to your needs. YOUR_NODE_URL should be filled in with a url for the network that you wish to deploy to and the `LongShortPairCreator` value should be substituted with the creator address on that same network. Added to that you should also choose the corresponding financial product library (FPL) that makes sense for your use case. The script has been pre-filled with the FPL for the covered call options
+
+All the required Mumbai addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/80001.json) however to give you a quick reference here are a few key contracts to use. 
+
+- `Mumbai LongShortPairCreator:` 0x6883FeB1c131F58C1Cd629289Da3dE0051d2aa0d
+- `BinaryOptionLongShortPairFinancialProductLibrary:` 0x2158C256b2d9B2b58D90D3ddA1b6a90d64498F7d
+- `CoveredCallLongShortPairFinancialProductLibrary:` 0xc19B7EF43a6eBd393446F401d1eCFac01B181ac0
+- `LinearLongShortPairFinancialProductLibrary:` 0x2aBf1Bd76655de80eDB3086114315Eec75AF500c
+- `RangeBondLongShortPairFinancialProductLibrary:` 0xb53A60f595EE2418be9F6057121EE77f0249AC28
+
+Before deploying, please note that the script will deploy to the network that your node URL is for. If using Infura for a Mumbai deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+Run the deployment script with your specific parameters.
+```bash
+node index.js --gasprice 1 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x6883FeB1c131F58C1Cd629289Da3dE0051d2aa0d --expirationTimestamp 1643678287 --collateralPerPair 1000000000000000000 --priceIdentifier ETHUSD --collateralToken 0xd0a1e359811322d97991e03f863a0c30c2cf029c --syntheticName "ETH 9000 USD Call [December 2021]" --syntheticSymbol ETHc9000-1221 --financialProductLibrary 0xc19B7EF43a6eBd393446F401d1eCFac01B181ac0
+```
+## Deploying to Polygon Mainnet
+
+The method to deploy onto mainnet is the exact same as with Mumbai, with a few key items to note. Your `url`, `lspCreatorAddress` and `financialProductLibrary` will need to be amended for use on Polygon mainnet. Since the library addresses are different between the networks. 
+
+All the required Polygon Mainnet addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json) however to give you a quick reference here are a few key contracts to use. 
```

Comment:
Same thing about hardcoding contracts. I think this is just more confusing - since it is possible we will not have these always be up to date and should just have this github reference

Reply:
also changed